core/modules/search/search.module

```diff
@@ -418,8 +418,8 @@ function search_simplify($text) {
   // can't match higher UTF-8 characters as the leftmost bit of those are 1.
   $text = preg_replace('/[.-]{2,}/', ' ', $text);
 
-  // The dot, underscore and dash are simply removed. This allows meaningful
-  // search behavior with acronyms and URLs. See unicode note directly above.
+  // The dot, underscore and dash are removed. This allows meaningful search
```

Comment:
```suggestion
  // The dot, underscore, and dash are removed. This allows meaningful search
```

Reply:
The final comma is optional. Depends if we follow the "Oxford comma".